chapters/research-design.tex

```diff
@@ -1,288 +1,593 @@
-for research teams do not need track individuals over time,
-or analyze attrition and follow-up other than non-response.
-Cross-sectional designs can have a time dimension;
-they are then called ``repeated cross-sections'',
-but do not imply a panel structure for individual observations.
-
-Typically, the cross-sectional model is developed
-only with controls for the research design.
-\textbf{Balance checks}\sidenote{\url{https://dimewiki.worldbank.org/wiki/iebaltab}} can be utilized, but an effective experiment
-can use \textbf{stratification} (sometimes called blocking) aggressively\sidenote{\url{https://blogs.worldbank.org/impactevaluations/impactevaluations/how-randomize-using-many-baseline-variables-guest-post-thomas-barrios}} to ensure balance
-before data is collected.\cite{athey2017econometrics}
-\index{balance}
-Stratification disaggregates a single experiment to a collection
-of smaller experiments by conducting randomization within
-``sufficiently similar'' strata groups.
-Adjustments for balance variables are never necessary in RCTs,
+When it is an RCT, a randomization process constructs the control group at random
+from the population that is eligible to receive each treatment.
+When it is observational, we present other evidence that a similar equivalence holds.
+Therefore, by construction, each unit's receipt of the treatment
+is unrelated to any of its other characteristics
+and the ordinary least squares (OLS) regression
+of outcome on treatment, without any control variables,
+is an unbiased estimate of the average treatment effect.
+Cross-sectional data is simple to handle because
+for research teams do not need track anything over time.
+A cross-section is simply a representative set of observations
+taken at a single point in time.
+If this point in time is after a treatment has been fully delivered,
+then the outcome values at that point in time
+already reflect the effect of the treatment.
+
+What needs to be carefully maintained in data for cross-sectional RCTs
+is the treatment randomization process itself,
+as well as detailed field data about differences
+in data quality and loss-to-follow-up across groups.\cite{athey2017econometrics}
+Only these details are needed to construct the appropriate estimator:
+clustering of the estimate is required at the level
+at which the treatment is assigned to observations,
+and controls are required for variables which
+were used to stratify the treatment (in the form of strata fixed effects).\sidenote{
+  \url{https://blogs.worldbank.org/impactevaluations/impactevaluations/how-randomize-using-many-baseline-variables-guest-post-thomas-barrios}}
+\textbf{Randomization inference} can be used
+to esetimate the underlying variability in the randomization process
+(more on this in the next chapter).
+\textbf{Balance checks} are often reported as evidence of an effective randomization,
+and are particularly important when the design is quasi-experimental
```

Comment:
Should I remove that from this section? I really don't want to list out for each design how these things differ since that is a general idea addressed in the previous and not intended to be the point of this section

Reply:
moved to issue #337